.drone.yml

```diff
@@ -131,25 +130,22 @@ build:
       repo: vmware/vic
       event: [ push, tag ]
       branch: [ master, releases/*, refs/tags/* ]
-      success: true
+      status: success
 
-notify:
-  slack:
-    webhook_url: $$SLACK_URL
-    channel: mwilliamson-staff
```

Comment:
The channel is not required. It is a part of the slack webhook https://vmware.slack.com/services/B59S91JFR